guide/advice-for-new-collaborators.rst

```diff
@@ -14,15 +16,15 @@ Goals of using DART
 -------------------
 
 DART is the Data Assimilation Research Testbed.  It is a collection of 
-tools and routines and scripts that allow users to built custom solutions
+tools and routines and scripts that allow users to build custom solutions
 and explore a variety of DA related efforts.  It is not a turnkey system;
 it must be built before use and is often customized based on needs and goals.
 
 DART is often used for the following types of projects:
 
 - Learning about Data Assimilation (DA)
 - Using DART with an existing model and supported observations
```

Comment:
It would be good to link to the list of supported models and observations

These pages:
supported models: https://docs.dart.ucar.edu/en/latest/models/README.html
supported observations: https://docs.dart.ucar.edu/en/latest/observations/obs_converters/README.html#converter-programs